src/lib/eslint/JavascriptEslintStrategy.ts

```diff
@@ -0,0 +1,72 @@
+import { EslintStrategy } from './BaseEslintEngine';
+import {TYPESCRIPT_RULE_PREFIX} from '../../Constants';
+import {Config} from '../util/Config';
+import {Controller} from '../../ioc.config';
+import { Logger } from '@salesforce/core';
+
+const ES_CONFIG = {
+	"extends": ["eslint:recommended"],
+	"parserOptions": {
+		"sourceType": "module",
+		"ecmaVersion": 2018,
+	},
+	"ignorePatterns": [
+		"node_modules/!**"
+	],
+	"useEslintrc": false // TODO derive from existing eslintrc if found and desired
+};
+
+export class JavascriptEslintStrategy implements EslintStrategy {
+	private static ENGINE_NAME = "eslint";
+	private static LANGUAGE = ["javascript"];
+
+	private initialized: boolean;
+	protected logger: Logger;
+	private config: Config;
+
+	public async init(): Promise<void> {
+		if (this.initialized) {
+			return;
+		}
+		this.logger = await Logger.child(this.getName());
+		this.config = await Controller.getConfig();
+		this.initialized = true;
+	}
+
+	public isEnabled(): boolean {
+		return this.config.isEngineEnabled(this.getName());
+	}
+
+	getLanguage(): string[] {
```

Comment:
If this is `getLanguage`, why is it returning an array?

Reply:
Yeah, that's awkward. Let me fix the names.